.infrastructure/12_certs.tf

```diff
@@ -0,0 +1,31 @@
+resource "aws_acm_certificate" "cms" {
```

Comment:
I'd try with [the module](https://registry.terraform.io/modules/terraform-aws-modules/acm/aws/latest) here as well. what do you think?

Reply:
with last commit is used records and acm modules